Lesson34/selcan/lesson34-app/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,40 @@
+import { useContext } from "react";
+import "./styles.css";
+import {
+  UserContext,
+  UserDispatchContext,
+  UserActionTypes,
+} from "../..//userContext.jsx";
+
+export const Navbar = () => {
+  const User = useContext(UserContext);
```

Comment:
```suggestion
  const user = useContext(UserContext);
```

This is a variable, not a component name so it should be in lowercase.